roles/benchmark/tasks/run.yml

```diff
@@ -17,6 +17,7 @@
   run_once: true
   vars:
     operation: run
+    hyperfoil_run_description: "{{ server_image }}-{{ cache_file }}"
```

Comment:
This isn't descriptive enough, but I guess it is probably fine as a default value designed to be overridden by the user.